frontend/public/components/nav.jsx

```diff
@@ -325,6 +325,7 @@ export class Nav extends React.Component {
             <ResourceNSLink resource="configmaps" name="Config Maps" onClick={this.close} />
             <ResourceNSLink resource="secrets" name="Secrets" onClick={this.close} />
             <ResourceNSLink resource="resourcequotas" name="Resource Quotas" onClick={this.close} />
+            <ResourceNSLink resource="horizontalpodautoscalers" name="Horizontal Pod Autoscalers" onClick={this.close} />
```

Comment:
I tend to agree "HPAs" is better here. We've already abbreviated "CRDs", so it wouldn't be the first.